core/src/main/java/org/mobilitydata/gtfsvalidator/util/UrlUtil.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.mobilitydata.gtfsvalidator.util;
+
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.MalformedURLException;
+import java.net.URL;
+import org.apache.commons.validator.routines.UrlValidator;
+
+/** Utility functions for validating urls */
+public class UrlUtil {
+  private static final int URL_ERROR_CODE = 404;
```

Comment:
The `404` error code seems to be too strict to me. I think we can be more open as `>=200 and <=399` to allow services to respond in a valid range. This will cover cases for services with invalid `5XX` and `1XX` responses.

Reply:
Done!